okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/IoExceptionResponseHandler.java

```diff
@@ -44,7 +44,7 @@
                     : "<empty>";
 
             return Optional.of(new SafeIoException(
-                    String.format("Error %s. (Failed to parse response body as SerializableError.)", response.code()),
+                    "Failed to parse response body as SerializableError",
```

Comment:
This needed to change as a result of upgrading `safe-logging` which requires these message to be compile-time constants.

Reply:
👍

Reply:
This broke clients that previously tried parsing the error code.